src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double katetA;
+    private final double katetB;
+    private String color;
+
+    public RightTriangle(double katetA, double katetB, String color) {
+        this.katetA = katetA;
+        this.katetB = katetB;
+        setColor(color);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + getArea()
+                + " sq. units, katet A: " + katetA + " units, katet B: " + katetB
+                + " units, color: " + getColor());
+    }
+
+    @Override
+    public double getArea() {
+        return (katetA * katetB) / 2;
```

Comment:
Consider using more descriptive variable names for `katetA` and `katetB`. In the context of a right triangle, it's common to refer to the sides as 'legA' and 'legB' or 'base' and 'height', which might be more intuitive for someone reading the code.